tests/test_is_element_in_list.py

```diff
@@ -0,0 +1,5 @@
+from src.is_element_in_list import is_element_in_list
+
+def test_is_element_in_list() -> None:
+    assert is_element_in_list([1,2,3], 2) == True
+    assert is_element_in_list([1,2,3], 4) == False
```

Comment:
Definisci l'arry ay di elementi `[1,2,3]` in una costante

Reply:
ho parametrizzato tutto, va bene così?
 [commit](https://github.com/UNICT-Quality-Development/qd-starter-pack/pull/172/commits/1236eaf8ff23b51192d18800ae9f774ba56bcbac)